src/main/java/controller/BlackjackController.java

```diff
@@ -0,0 +1,128 @@
+
+    public BlackjackController(OutputView outputView, InputView inputView) {
+        this.outputView = outputView;
+        this.inputView = inputView;
+    }
+
+    public void run() {
+        List<String> playerNames = Parser.parseStringToList(inputView.inputUsers());
+        List<Player> users = playerNames.stream().map(Player::new).toList();
+        GameManger gameManger = new GameManger(users, new Dealer(), new CardDeck());
+        distributionFirstCard(gameManger, playerNames);
+
+        additionalPlayerCard(gameManger, playerNames);
+        additionalDealerCard(gameManger);
+
+        createGameResult(gameManger, playerNames);
+        calculateGameResult(gameManger);
+
+        inputView.close();
+    }
+
+    private void distributionFirstCard(GameManger gameManger, List<String> playerNames) {
+        gameManger.firstHandOutCard();
+
+        displayOpenCard(gameManger.getDealer().getName(), gameManger.getDealer());
+        playerNames.forEach(playerName -> displayOpenCard(playerName, gameManger.findUserByUsername(playerName)));
+    }
+
+    private void additionalDealerCard(GameManger gameManger) {
+        User dealer = gameManger.getDealer();
+        while (!dealer.isImpossibleDraw()) {
+            dealer.drawCard(gameManger.handOutCard());
+            outputView.displayDealerAddCard();
+        }
+    }
+
+    private void additionalPlayerCard(GameManger gameManger, List<String> playerNames) {
+        playerNames.forEach(playerName ->
+                addCardAllPlayer(gameManger, playerName));
+    }
+
+    private void calculateDealerResult(Map<User, GameResult> gameResult) {
+        int dealerLoseCount = getResultStateCount(gameResult, GameResult.WIN);
+        int dealerWinCount = getResultStateCount(gameResult, GameResult.LOSE);
+        int dealerDrawCount = getResultStateCount(gameResult, GameResult.DRAW);
+
+        outputView.displayDealerGameResult(dealerLoseCount, dealerWinCount, dealerDrawCount);
+    }
+
+    private int getResultStateCount(Map<User, GameResult> gameResult, GameResult status) {
+        return (int) gameResult.entrySet().stream()
+                .filter(entry -> entry.getValue() == status)
+                .count();
+    }
+
+    private void addCardAllPlayer(GameManger gameManger, String playerName) {
+        User player = gameManger.findUserByUsername(playerName);
+        while (!player.isImpossibleDraw()) {
+            String yesOrNo = inputView.inputYesOrNo(playerName);
+            if (yesOrNo.equalsIgnoreCase("N")) {
+                return;
+            }
+            player.drawCard(gameManger.handOutCard());
+            displayOpenCard(playerName, player);
+        }
+    }
+
+    private void displayOpenCard(String playerName, User player) {
+        List<String> cards = player.openInitialCard().stream()
+                .map(card -> CardConverter.createTrumpCard(card.cardShape(), card.cardNumber())).toList();
+        outputView.displayOpenCards(playerName, cards);
```

Comment:
`CardConverter`를 View에 두신걸로 봤을 때,   Card를 노출하기위한 변환은 View에서 수행해도 될 것 같은데요. 😃
OutputView에서 이 변환 로직을 수행하면 어떨까요?  (가능하면 Controller에서 CardConverter를 통해서 변환하고있는 부분들은 모두 OutputView에서 수행하도록 해주세요 🙏)

```suggestion
        outputView.displayOpenCards(playerName, player.openInitialCard());
```